src/api/db/migrate/20211110142710_create_workflow_runs.rb

```diff
@@ -0,0 +1,15 @@
+class CreateWorkflowRuns < ActiveRecord::Migration[6.1]
+  def change
+    create_table :workflow_runs, id: :integer do |t|
+      t.text :request_headers
+      t.text :request_payload
+      t.string :status
+      t.text :response
```

Comment:
This would be only for the response body, right?

```suggestion
      t.text :response_body
```

Reply:
Exactly. Changed.